.github/workflows/issue_stale_bot.yml

```diff
@@ -16,16 +17,16 @@ jobs:
       - uses: actions/stale@v4
         with:
           stale-issue-message: '
-          :exclamation: This issue is stale because it has been open for 20 days with no activity.
+          :exclamation: This issue is stale because it has been open for 60 days with no activity.
           
-          Remove `S2-stale` label or update it, otherwise this issue will be closed in 5 days.
+          Remove `S2-stale` label or update it, otherwise this issue will be closed in 7 days.
```

Comment:
Would it be possible to comment again 1 day before it's closed? Otherwise, it'll be closed silently if the first stale warning is neglected.

Reply:
> Would it be possible to comment again 1 day before it's closed? Otherwise, it'll be closed silently if the first stale warning is neglected.

I believe the original action doesn't support this, as the first warning should already be handled correctly 😂 
If we need it we could implement it ourselves though, simply detect this comment and write another comment in 6 days.

Reply:
I see. Just a good-to-have feature. Let's see if we really need it later.